README.md

```diff
@@ -301,11 +301,35 @@ request_headers: '{
 
 ## Events
 
-Events are a way to organize code. They allow you to add your logic to existing commands. They are executed asynchronously in the background.
+**Events** in platformOS allow you to record that something has happened in the system. Think of them as logs that capture these actions. When an event occurs, other parts of the application might need to react to it. These events are stored using GraphQL mutations and can include various arguments detailing the specifics of what happened. They enable you to add your logic to existing commands and are executed asynchronously in the background.
```

Comment:
```suggestion
**Events**allow you to record that something has happened in the system. Think of them as logs that capture these actions. When an event occurs, other parts of the application might need to react to it. In platformOS, these events are stored using GraphQL mutations and can include various arguments detailing what happened. They enable you to add logic to existing commands and are executed asynchronously in the background.
```

Events are generic concept, so move "in platformOS" when we discuss platformOS specific thing, which is events implementation :)